### PR TITLE
Various v3 editorial issues. 

### PR DESCRIPTION
**Pull request by Kirky-J**
Resolution of a number of editorial issues (formatting, typo's etc.) for v3.0. Includes fixes for:
- 154
- 159
- 153
- 152
- 140
- 139
- 138
- 137
- 136
- 102

### CODE REVIEW

File: NDcPP_v3_0.adoc [hunk 1]
```diff
@@ -2280,7 +2280,7 @@ _The following table provides the ST author with instructions for completing the
 
 _The option 'provides' shall be selected if the TOE provides the ability to configure the list of ciphers as defined in FCS_DTLSC_EXT.1.1 (e.g. enabling/disabling of ciphers, ordering, assigning priorities). Otherwise, the option 'does not provide' shall be selected. If the 'provides' option is selected by the ST Author, the ST Author also selects the corresponding function in FMT_SMF.1._
 
-*FCS_DTLSC_EXT.1.7* The TSF _shall prohibit the use of the following extensions:
+*FCS_DTLSC_EXT.1.7* The TSF shall prohibit the use of the following extensions:
```
Comment (kr15tyk) [inline NDcPP_v3_0.adoc:2283]: @Kirky-J FCS_DTLSS_EXT.1.9, FCS_TLSC_EXT.1.7, FCS_TLSS_EXT.1.6 has the same wording italicized.

File: NDcPP_v3_0.adoc [hunk 2]
```diff
@@ -2309,7 +2309,7 @@ _The use of out-of-band provisioned pre-shared keys creates a potential security
 
 *_Application Note {counter:appnote_count}_*
 
-_The ciphersuites to be tested in the evaluated configuration are limited by this requirement and must be selected from the ciphersuites defined in List 1, chap. B.3.1.6. The ST author should select the ciphersuites that are supported. Even though RFC 5246 and RFC 6347 mandate implementation of specific ciphers, this cPP depreciates use of SHA-1. Support for TLS_RSA_WITH_AES_128_CBC_SHA is not claimed in order to claim conformance to this cPP._
+_The ciphersuites to be tested in the evaluated configuration are limited by this requirement and must be selected from the ciphersuites defined in List 1 or List 2, chap. B.3.1.5. The ST author should select the ciphersuites that are supported. Even though RFC 5246 and RFC 6347 mandate implementation of specific ciphers, this cPP deprecates use of SHA-1. Support for TLS_RSA_WITH_AES_128_CBC_SHA is not claimed in order to claim conformance to this cPP._
```
Comment (kr15tyk) [inline NDcPP_v3_0.adoc:2312]: @Kirky-J Should the wording in the first sentence in paragraphs 63, 70, 96, and 104 be consistent?

File: NDcPP_v3_0.adoc [hunk 3]
```diff
@@ -3049,7 +3049,7 @@ _Although this is out-of-scope of this cPP, it is strongly advised to protect al
 
 *FPT_TUD_EXT.2.1* The TSF shall check the validity of the code signing certificate before installing each update.
 
-*FPT_TUD_EXT.2.2* *If revocation information is not available for a certificate in the trust chain that is not a trusted certificate designated as a trust anchor, the TSF shall [selection: _not install the update, allow the Administrator to choose whether to accept the certificate in these cases_].*
+*FPT_TUD_EXT.2.2* If revocation information is not available for a certificate in the trust chain that is not a trusted certificate designated as a trust anchor, the TSF shall [selection: _not install the update, allow the Administrator to choose whether to accept the certificate in these cases_].
```
Comment (kr15tyk) [inline NDcPP_v3_0.adoc:3052]: @Kirky-J Bold also needs to be corrected in Section C.4.4.1.

**Review by kr15tyk — CHANGES_REQUESTED** [review]
See notes for recommended edits.

File: NDcPP_v3_0.adoc [hunk 4]
```diff
@@ -2164,15 +2164,15 @@ Due to the structural differences between TLS 1.2 on the one hand and TLS 1.3 on
 
 *_Application Note {counter:appnote_count}_*
 
-_The ciphersuites to be tested in the evaluated configuration are limited by this requirement requirement and shall be selected from the ciphersuites defined in List 1 (DTLS 1.2) and List 2 (DTLS 1.3), respectively (both in chap. B.3.1.6). The ST author should select the ciphersuites that are supported. Even though RFC 5246 and RFC 6347 mandate implementation of specific ciphers, this cPP deprecates use of SHA-1. Support for TLS_RSA_WITH_AES_128_CBC_SHA is not claimed in order to claim conformance to this cPP._
+_The ciphersuites to be tested in the evaluated configuration are limited by this requirement requirement and shall be selected from the ciphersuites defined in List 1 (DTLS 1.2) and List 2 (DTLS 1.3), respectively (both in chap. B.3.1.5). The ST author should select the ciphersuites that are supported. Even though RFC 5246 and RFC 6347 mandate implementation of specific ciphers, this cPP deprecates use of SHA-1. Support for TLS_RSA_WITH_AES_128_CBC_SHA is not claimed in order to claim conformance to this cPP._
```
[the remark this answers is not in the document]
Comment (Kirky-J) [reply NDcPP_v3_0.adoc:2167]: Thanks and fixed.

File: NDcPP_v3_0.adoc (hunk 1 above)
Comment (Kirky-J) [reply NDcPP_v3_0.adoc:2283]: Removed incorrect formatting on these SFRs. Thanks!

File: NDcPP_v3_0.adoc (hunk 3 above)
Comment (Kirky-J) [reply NDcPP_v3_0.adoc:3052]: Nice catch. Thanks. Fixed.

File: NDcPP_v3_0.adoc (hunk 2 above)
Comment (Kirky-J) [reply NDcPP_v3_0.adoc:2312]: Adjusted the first sentences in App notes 62 and 70 to be consistent (DTLS) and 96 and 104 to be consistent (TLS).

**Comment by Kirky-J** [comment]
All suggested changes applied.

File: NDcPP_v3_0.adoc (hunk 1 above)
Comment (kr15tyk) [reply NDcPP_v3_0.adoc:2283]: @Kirky-J There's another instance of FCS_DTLSS_EXT.1.9, FCS_TLSC_EXT.1.7, FCS_TLSS_EXT.1.6 that still needs the italics fixed. Let me know if you want me to make the change.

Comment (Kirky-J) [reply NDcPP_v3_0.adoc:2283]: Ok. Got them now too!

**Review by kr15tyk — APPROVED** [review]

**Pull request merged**